src/web/mjs/connectors/templates/NovelCool.mjs

```diff
@@ -0,0 +1,88 @@
+import Connector from '../../engine/Connector.mjs';
+import Manga from '../../engine/Manga.mjs';
+
+export default class Novelcool extends Connector {
+
+    constructor() {
+        super();
+        super.id = undefined;
+        super.label = undefined;
+        this.tags = undefined;
+        this.url = undefined;
+
+        this.manga_selector = 'div.book-list div.book-info a';
+        this.manga_page_query = '/category/index_{page}.html'; // {page} will get replaced with the current page number
+        this.chapter_selector = 'div.chapter-item-list a';
+        this.page_selector = 'select.sl-page';
+        this.image_selector = 'div.pic_box source.manga_pic';
+        this.uri_title_selector = 'div.bookinfo-info h1.bookinfo-title';
+        this.text_novel_selector = 'p.chapter-start-mark';
+    }
+
+    async _getMangas() {
+        let more_pages = true;
+        let page = 1;
+
+        let mangas = [];
+        while (more_pages) {
+            let request = new Request(new URL(this.manga_page_query.replace('{page}',page), this.url), this.requestOptions);
+            let data = await fetch(request);
+
+            if ( data.url === this.url+this.manga_page_query.replace('{page}',page++) ) {
```

Comment:
Seems its the same as comparing to already available `request.url`

Reply:
How did i miss that? So obvious now.